scripts/requirements-fixed.txt

```diff
@@ -125,6 +125,7 @@ tomli==2.0.1              # via nrf-regtool
 tomlkit==0.12.1           # via pylint
 typed-ast==1.5.5          # via -r nrf/scripts/requirements-ci.txt
 typing-extensions==4.8.0  # via mypy, pyocd, python-can, qrcode, setuptools-scm, svada
+unidiff==0.7.5            # via internet
```

Comment:
just skip via comment. `via` comments are generated with pip-compile tool, but it is disabled at this moment

Reply:
ok, I will remove the comment :)